src/main/java/com/_up/megastore/controllers/interfaces/ISizeController.java

```diff
@@ -18,4 +18,6 @@ public interface ISizeController {
     @PutMapping (value = "/{sizeId}") @ResponseStatus(HttpStatus.OK)
     SizeResponse updateSize(@PathVariable UUID sizeId, @RequestBody @Valid UpdateSizeRequest updateSizeRequest);
 
+    @DeleteMapping (value = "/{sizeId}") @ResponseStatus(HttpStatus.OK)
+    void deleteSize(@PathVariable UUID sizeId);
```

Comment:
Esto como lo expliqué en otro review esto debería devolver un NO_CONTENT ya que no retornamos ningun payload

```suggestion
    @DeleteMapping (value = "/{sizeId}") @ResponseStatus(HttpStatus.NO_CONTENT)
    void deleteSize(@PathVariable UUID sizeId);
```